lib/rouge/lexers/csvs.rb

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class CSVS < RegexLexer
+      tag 'csvs'
+      title "csvs"
+      desc 'The CSV Schema Language (http://digital-preservation.github.io/csv-schema/)'
+      mimetypes 'text/x-csvssrc'
+      filenames '*.csvs'
+
+      def self.detect?(text)
+        return true if text.shebang? 'csvs'
```

Comment:
It can really be used in scripts? To be clear, you don't need a `self.detect?` method. It's fine to leave this out if it's not relevant for this lexer.

Reply:
Geez, what was I even trying to say here. I think somehow the beginning of this got cut off somehow. I meant to write:

> Can CSVS really be used with a shebang? To be clear, you don't need a `self.detect?` method. It's fine to leave this out if it's not relevant for this lexer.